src/govuk/common.mjs

```diff
@@ -6,6 +6,10 @@ import './vendor/polyfills/Element/prototype/closest.mjs'
  * TODO: Ideally this would be a NodeList.prototype.forEach polyfill
  * This seems to fail in IE8, requires more investigation.
  * See: https://github.com/imagitama/nodelist-foreach-polyfill
+ *
+ * @param {NodeListOf<Element>} nodes - NodeList from querySelectorAll()
+ * @param {(value: Element, key: number, parent: NodeListOf<Element>) => void} callback - Callback function to run for each node
```

Comment:
Where's this JSDoc block from @romaricpascal?

Reply:
Ahh I see, that's your suggestion

Because we've set `mode: 'typescript'` for compatibility with **node_modules** that export type declarations, maybe for the `jsdoc` CLI you'll also need to enable the TypeScript plugin?

But it's simple enough to use `@callback` so pushed another commit up 👍

Reply:
Forgot to update this comment

I've added **JSDoc generation** via https://github.com/alphagov/govuk-frontend/pull/2920 to confirm it all works

Type declarations like `import('Puppeteer').Page` are handled by the [`jsdoc-tsimport-plugin`](https://www.npmjs.com/package/jsdoc-tsimport-plugin) plugin

Reply:
Ahh sorry @36degrees that was @romaricpascal's suggestion too, it's a `@callback` now 🙌 

To answer your question though I grabbed the names from Microsoft's `forEach` definition:

https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L10079
```ts
interface NodeListOf<TNode extends Node> extends NodeList {
    item(index: number): TNode;
    /**
     * Performs the specified action for each node in an list.
     * @param callbackfn  A function that accepts up to three arguments. forEach calls the callbackfn function one time for each element in the list.
     * @param thisArg  An object to which the this keyword can refer in the callbackfn function. If thisArg is omitted, undefined is used as the this value.
     */
    forEach(callbackfn: (value: TNode, key: number, parent: NodeListOf<TNode>) => void, thisArg?: any): void;
    [index: number]: TNode;
}
```